Sources/FoundationEssentials/String/StringProtocol+Essentials.swift

```diff
@@ -91,6 +91,14 @@ extension UInt16 {
 
 // These provides concrete implementations for String and Substring, enhancing performance over generic StringProtocol.
 
+#if !FOUNDATION_FRAMEWORK
+@_spi(SwiftCorelibsFoundation)
+dynamic public func _cfStringEncodingConvert(string: String, using encoding: UInt, allowLossyConversion: Bool) -> Data? {
```

Comment:
Should we add a similar up-call for the bytes --> `String` direction so that we support decoding as well?

Reply:
Good idea, done.